setup.py

```diff
@@ -709,6 +709,7 @@ def is_package_excluded(package: str, exclusion_list: List[str]):
     'lazy_object_proxy~=1.3',
     'lockfile>=0.12.2',
     'markdown>=2.5.2, <3.0',
+    'markupsafe>=1.1.1, <2.0',
```

Comment:
It is not a new dependency as it was already required by jinja2, Flask-App builder etc

https://github.com/dpgaspar/Flask-AppBuilder/blob/master/requirements.txt